version.php

```diff
@@ -2,7 +2,8 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2023013100;
-$plugin->requires   = 2018120302; // Moodle v3.6.2             
-$plugin->release    = 'v1.0.0.23';
+/** @var object $plugin */
 $plugin->component = 'assignsubmission_pxaiwriter';
+$plugin->version   = 2023040300;
+$plugin->requires   = 2018120302; // Moodle v3.6.2
+$plugin->release    = '1.0.1 (Build: 2023-03-30)';
```

Comment:
$plugin->release should be `v1.1.0 (Build: 2023-04-03)` to match the version and README release note

Reply:
Done :)